src/GraphQL/Authorization/GraphQLAuthorizeAttribute.cs

```diff
@@ -4,10 +4,17 @@
 namespace GraphQL
 {
     /// <summary>
-    /// Attribute to apply authorization policy when using schema first approach.
```

Comment:
> when using schema first approach.

No only schema first now?

Reply:
No it is not.  It is used for "type-first" (i.e. the auto-registering object graph type) also.